modules/serverless-deleting-subscriptions-kn.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+//  * serverless/event_workflows/serverless-subs.adoc
+
+[id="serverless-deleting-subscriptions-kn_{context}"]
+= Deleting subscriptions using the Knative CLI
+
+You can delete a subscription by using the `kn` CLI.
+
+.Procedure
+
+* Delete a subscription:
++
+[source,terminal]
+----
+kn subscription delete <subscription_name>
```

Comment:
```suggestion
$ kn subscription delete <subscription_name>
```
The style is a different from other commands. Is it missing `$` sign?

Reply:
It is, whoops! Thank you 🙂